pages/glossary/distinguishable-box-shadows.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Distinguishable box-shadows
+key: distinguishable-box-shadows
+unambiguous: true
+objective: true
+---
+
+Look at the [computed](https://drafts.csswg.org/css-cascade/#computed-value) values of the `box-shadow` style property.
```

Comment:
The following should probably pass:

```
<p style="box-shadow: 3px 3px 3px black;">
   Hello <a href="/" style="box-shadow: 3px 3px 3px black;">World</a>.
<p>
```

Even if the shadows are the same, the fact that one is drawn around the inline element and the other is around the block should give away that something is different.

Reply:
Truth be told, I have doubts about this example. When I've seen it rendered (after removing the link color and underline) my reaction is that whatever happens when I click "World" would also happen when I clicked the paragraph. To me it conveyed the same "behaviour". 

What are others opinions on this?